activemodel/lib/active_model/attribute_registration.rb

```diff
@@ -30,33 +40,39 @@ def attribute_types # :nodoc:
       end
 
       private
-        class PendingAttribute # :nodoc:
-          attr_accessor :type, :default
-
-          def apply_to(attribute)
-            attribute = attribute.with_type(type || attribute.type)
-            attribute = attribute.with_user_default(default) if defined?(@default)
-            attribute
+        PendingType = Struct.new(:name, :type) do # :nodoc:
```

Comment:
Avoid use structs because it is harder to search for it. `class PendingType` will not return the definition of this class.